src/networking.c

```diff
@@ -3677,6 +3698,70 @@ static int parseClientFiltersOrReply(client *c, int index, clientFilter *filter)
                 return C_ERR;
             }
             index += 2;
+        } else if (!strcasecmp(c->argv[index]->ptr, "idle") && moreargs) {
+            long long tmp;
+
+            if (getLongLongFromObjectOrReply(c, c->argv[index + 1], &tmp,
+                                             "idle is not an integer or out of range") != C_OK)
+                return C_ERR;
+            if (tmp <= 0) {
+                addReplyError(c, "idle should be greater than 0");
+                return C_ERR;
+            }
+
+            filter->idle = tmp;
+            index += 2;
+        } else if (!strcasecmp(c->argv[index]->ptr, "flags") && moreargs) {
+            filter->flags = c->argv[index + 1]->ptr;
+            if (validateClientFlagFilter(filter->flags) == C_ERR) {
+                addReplyError(c, "unknown flags found in the filter");
```

Comment:
Should we add the `filter->flags` to the error response for easier logging on the application side ?

```suggestion
                addReplyErrorFormat(c, "unknown flags found in the provided filter: %s", filter->flags);
```

Reply:
done